src/test/kotlin/ContainerUnitTest.kt

```diff
@@ -237,6 +238,60 @@ class ContainerUnitTest {
         }
     }
 
+    private val getContentFromRCCases = listOf(
+        "sng_book_various_content_rc",
+        "sng_book_various_content_rc.zip"
+    )
+
+    @Test
+    @Throws(Exception::class)
+    fun getContentFromRC() {
+        getContentFromRCCases.forEach {
+            val classLoader = this.javaClass.classLoader
+            val resource = classLoader.getResource(it)
+            val rcPath = File(resource!!.toURI().path)
```

Comment:
Don't do this; getResourceAsStream and them copy it to a temporary file.

This should also be done probably as a setup thing before the test is run.

Tests for getting content from an RC might as such be better done in their own test file